.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,21 +1,126 @@
 <component name="InspectionProjectProfileManager">
   <profile version="1.0" is_locked="false">
     <option name="myName" value="Project Default" />
-    <option name="myLocal" value="false" />
+    <inspection_tool class="AbsoluteAlignmentInUserInterface" enabled="true" level="WARNING" enabled_by_default="true" />
```

Comment:
I try to keep changes to settings out of commits. Could you remove the modification of the settings from the commit or is this necessary here?